src/routes/auth/providers/utils.ts

```diff
@@ -162,7 +163,7 @@ export const initProvider = <T extends Strategy>(
           {
             ...PROVIDERS[strategyName],
             ...options,
-            callbackURL: `${APPLICATION.SERVER_URL}/auth/providers/${strategyName}/callback`,
+            callbackURL: `http://139.59.154.242:3000/auth/providers/${strategyName}/callback`,
```

Comment:
@komninoschat was this only for testing?

Reply:
Oops. Nice catch!